### PR TITLE
DDT-1337: Added timestampID to TimestampTO

### PR DESCRIPTION
**Pull request by Abdiiir**
- Added test for duplicate timestampID

### CODE REVIEW

File: jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java [hunk 1]
```diff
@@ -98,4 +98,14 @@ public class OperationsEvent {
   @Column(name = "miles_to_destination_port")
   private Float milesToDestinationPort;
 
+  @Transient Boolean newRecord;
+  @Override
+  public boolean isNew() {
+    return this.newRecord;
```
Comment (nt-as) [inline jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java:104]:
```suggestion
    return eventID != null || this.newRecord;
```

**Comment by nt-as** [comment]
(also has a test failure)

File: jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java (hunk 1 above)
Comment (Abdiiir) [reply jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java:104]: will amend commit.

**Review by nt-as — CHANGES_REQUESTED** [review]
Please update the postman collection to post the same timestamp twice with the same ID (so it does not depend on operations event from SQL).

**Review by nt-gt — APPROVED** [review]

**Pull request merged**